### ISSUE TITLE
[Bug]: Footer is not responsive "Quick links-> elements are missing" . 

### ISSUE THREAD

**Issue by ganjivamshiii** [mansiruhil/AmbuFlow#514]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

Respnsiveness is missing in footer and also header.

### Add Screenshots

![Screenshot 2024-10-09 205758](https://github.com/user-attachments/assets/8b8ad046-0f48-4055-9daf-9a919e317dcd)

### What browsers are you seeing the problem on?

Chrome

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I'm a GSSOC contributor
- [X] I want to work on this issue
- [X] I'm willing to provide further clarification or assistance if needed.

**Comment by mansiruhil13** [comment]
yes in mobile mode they are missing, but in laptop chrome visible perfectly
